src/chainbin/nnet3-chain-get-egs.cc

```diff
@@ -30,6 +30,60 @@
 namespace kaldi {
 namespace nnet3 {
 
+/** add iVectors to Egs.
+*/
+static void GetIvectorsForEgs(int32 t, int32 left_context,
```

Comment:
please rename t with a more meaningful name.   e.g. call it chunk_start and call frames_per_eg chunk_size.